torch/_decomp/__init__.py

```diff
@@ -265,6 +265,7 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
             aten.arange.start,
             aten.avg_pool2d_backward,
             aten.baddbmm,
+            aten.special_bessel_j0,
```

Comment:
ATen core decomps shouldn't decompose to prims. We don't need it for lowering purposes either so lets just delete this from the list.

Reply:
Fixed.